src/components/radio-button/radio-button-group.spec.tsx

```diff
@@ -126,16 +126,9 @@ describe("RadioButtonGroup", () => {
   });
 
   describe("validations", () => {
-    it.each([
-      ["error", "string"],
-      ["error", true],
-      ["warning", "string"],
-      ["warning", true],
-      ["info", "string"],
-      ["info", true],
-    ])(
+    it.each(["error", "error", "warning", "warning", "info", "info"])(
```

Comment:
suggestion: should we just remove the duplicates?

I don't see much point in keeping them now since value has been removed

Reply:
That is a very valid point. Good spot. I'll get this suggestion implemented 👍🏻